030-Data-types/030-data-model.mdx

```diff
@@ -205,29 +205,34 @@ In the above, it's worth noting:
 
 ## Column types
 
-### string
+### text
 
-The `string` type represents a simple string. The values of this type are indexed both for quick exact matching and for full-text search. Set `unique` to ensure the strings you insert are unique. Set `notNull` to require a value.
+The `text` type represents a string or a long-form text. If a value must be provided, you can use the `notNull` setting to enforce this requirement.
 
 Example definition:
 
 ```json
 {
-  "name": "name",
-  "type": "string"
+  "name": "address",
+  "type": "text"
 }
 ```
 
-### text
+### string
 
-The `text` type represents a long-form text. Unlike the `string type`, `text` column values are indexed and optimized for efficient free-text searches, rather than exact matches. If a value must be provided, you can use the `notNull` setting to enforce this requirement.
+<Alert status="warning">
+  Note: The string type is deprecated. Existing string columns will continue working but this type is not available for
```

Comment:
(via the UI, they could still technically create one through the CLI)

Reply:
agreed, added fix